### DISCUSSION TITLE
Easy way to display month and year ?

### DISCUSSION THREAD

**Discussion by KhairulAzmi21** [omnia-digital/livewire-calendar#28] [Q&A] [answered]
![Screenshot 2024-03-24 at 12 34 07](https://github.com/omnia-digital/livewire-calendar/assets/12378983/18c6bd10-fbba-4428-8342-e5a0de69587e)

its there easier way to display current month and year ? 

right now i doing like this which I think dumb . 

```php
<div class="text-2xl font-semibold">{{ $monthGrid[1][7]->format('M Y') }}</div>
```

**Comment by joshtorres** [comment, marked as answer]
We've done it like this in the past
`<div>{{ $startsAt->englishMonth }} {{ $startsAt->year }}</div>`

**Reply by lrljoe** [reply]
Worth noting as well, that you may be localising Carbon globally, in which case:
```
    <div> {{ ucfirst($startsAt->isoFormat('MMMM YYYY')) }}</div>
```

Or you may want to localise it directly (for example - to French)
```
  <div> {{ ucfirst($startsAt->locale('fr_FR')->isoFormat('MMMM YYYY')) }}</div>
```
